polaris-react/src/components/Navigation/_variables.scss

```diff
@@ -56,7 +56,7 @@ $nav-animation-variables: (
     @include no-focus-ring;
   }
 
-  @include breakpoint-after(nav-min-window-corrected()) {
+  @include breakpoint-after($nav-min-window-corrected) {
```

Comment:
Do we need an `@import` for `/styles/common'` here or is it okay since there's an import in `polaris-react/src/components/Navigation/Navigation.scss`?

Reply:
> or is it okay since there's an import in polaris-react/src/components/Navigation/Navigation.scss

Exactly, we're good 👍